### ISSUE TITLE
unable to pull latest

### ISSUE THREAD

**Issue by paOol** [urbanmassage/docker-node#1]
```

$ docker pull urbanmassage/node
Using default tag: latest
Pulling repository docker.io/urbanmassage/node
Tag latest not found in repository docker.io/urbanmassage/node
```

**Comment by gileze33** [comment]
Oops, we only ever use the tags such as `5`, I'll add support for `latest` now

**Comment by gileze33** [comment]
Solved in 1b913769ac7a8a2cbd48f5d52f77bd16d53d4d56
